addons/survey/static/src/js/survey_form.js

```diff
@@ -133,22 +145,134 @@ publicWidget.registry.SurveyFormWidget = publicWidget.Widget.extend({
             $("html, body").animate({ scrollTop: 0 }, "fast");
         }
         else if (result && result.fields && result.error === 'validation') {
-            var fieldKeys = _.keys(result.fields);
-            _.each(fieldKeys, function (key) {
-                self.$("#" + key + '>.o_survey_question_error').append($('<p>', {text: result.fields[key]})).removeClass('d-none');
-                if (fieldKeys[fieldKeys.length - 1] === key) {
-                    self._scrollToError(self.$('.o_survey_question_error:visible:first').closest('.js_question-wrapper'));
-                }
-            });
+            self.$('.o_survey_form_content').fadeIn(400);
```

Comment:
As said in previous review, I would not bother fading in if there are validation errors.
(If it's easier to call fadein anyway, maybe call it with a `0` parameter?)